frontend/shared/src/components/forms/inputs/Checkbox.tsx

```diff
@@ -0,0 +1,67 @@
+import { Checkbox as HdsCheckbox } from 'hds-react';
+import noop from 'lodash/noop';
+import React from 'react';
+import {
+  Controller,
+  ControllerRenderProps,
+  Path,
+  useFormContext,
+} from 'react-hook-form';
+import useToggle from 'shared/hooks/useToggle';
+import InputProps from 'shared/types/input-props';
+
+const Checkbox = <T,>({
```

Comment:
```suggestion
const Checkbox = <T>({
```

Reply:
That's silly but needed in tsx files because otherwise the ts compiler thinks that it's an element.